### PR TITLE
Unified the getter flag of the signer's private key to "from"

### PR DESCRIPTION
**Pull request by psy2848048**
#### Before
  * Key is specified by `--wallet`, `--address`, and `--nickname` flags

#### After
  * Unified that to `--from`

#### Searching rule
  1. No `--from`
    - Check the # of the local wallet
      * If only 1, use this
      * If none or multiple, raise error
  1. With `--from`
    - Search by following priority:
      1. Local wallet alias
      1. Address parsing
      1. Querying to nickname module

#### Modified logic
* Modified address shipping rule in client context
* Modified key selection rule in each CLI
* Reduced recurring code and put it into util

### How to review

  * `clif nickname set princesselsa --from elsa`
  * `clif nickname set princesselsa` if only `elsa` exists
  * `clif nickname change-to princesselsa fridayxxxxxxxx... --from elsa`
  * `clif hdac transfer-to princessanna 1000000 100000000 25000000 --from elsa`
  * `clif hdac transfer-to princessanna 1000000 100000000 25000000` if only `elsa` exists
  * `clif hdac bond/unbond 10000000000 1000000000 50000000 --from elsa`
  * `clif hdac bond/unbond 10000000000 1000000000 50000000` if only `elsa` exists
  * `clif hdac create-validator --from elsa --pubkey fridayvalconspubxxxxx --moniker node1`
  * `clif hdac create-validator --pubkey fridayvalconspubxxxxx --moniker node1` if only `elsa` exists

Multinode test has been passed. [Test log](http://132.145.81.228:8080/job/05_(Do%20not%20trigger%20yourself)%20Test%20run/36/console)

### CODE REVIEW

File: x/executionlayer/client/util/util.go [hunk 1]
```diff
@@ -64,3 +65,46 @@ func ProtobufSafeDecodeingHexString(str string) ([]byte, error) {
...
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse into address, or no registered address of the given nickname '%s'", valueFromFromFlag)
+		}
+		key, err = kb.GetByAddress(addr)
+		if err != nil {
+			return nil, err
+		}
+		return key, nil
+	}
+
```
Comment (JoowonYun) [inline x/executionlayer/client/util/util.go]: I think getbalnace should be possible without keys.
In the above case, it is impossible to search for an address that does not have a key.

```$ clif hdac getbalance --from friday1lgharzgds89lpshr7q8kcmd2esnxkfpwmfgk32```

Comment (psy2848048) [reply x/executionlayer/client/util/util.go]: Damm, I did it too automatically

https://github.com/hdac-io/friday/pull/77/commits/b790e50f48a5acb0ade2d10970fa947ce67755ee

Comment (psy2848048) [reply x/executionlayer/client/util/util.go]: Input key searching logic added if the input is not valid address nor registered nickname.

https://github.com/hdac-io/friday/pull/77/commits/96c8383242f9942b84422521489c5d8f522af74f

[Test log](http://132.145.81.228:8080/job/05_(Do%20not%20trigger%20yourself)%20Test%20run/37/console)

**Comment by psy2848048** [comment]
Thank you for your 2 confirms.
This push will be merged after #78 merge & rebase

**Comment by psy2848048** [comment]
Close this and reopened as #80

**Pull request closed without merging**